packages/in_app_purchase/in_app_purchase_android/lib/src/in_app_purchase_android_platform_addition.dart

```diff
@@ -146,4 +146,15 @@ class InAppPurchaseAndroidPlatformAddition
       (BillingClient client) => client.isFeatureSupported(feature),
     );
   }
+
+  /// Returns Play billing country code based on ISO-3166-1 alpha2 format.
+  ///
+  /// See: https://developer.android.com/reference/com/android/billingclient/api/BillingConfig
+  /// See: https://unicode.org/cldr/charts/latest/supplemental/territory_containment_un_m_49.html
+  Future<String> getCountryCode() async {
```

Comment:
I think method should get tests added to `in_app_purchase_android_platform_addition_test.dart` to cover